.changes/next-release/bugfix-AmazonCloudfront-1fa6f75.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "Amazon Cloudfront",
+    "contributor": "",
+    "description": "Decouple policy logic from resource url for getSignedUrlWithCustomPolicy"
```

Comment:
This seems to be implementation detail.

Suggesting: 
```
Allow users to specify resource URL pattern in `CloudFrontUtilities#getSignedUrlWithCustomPolicy`. See [#5577](https://github.com/aws/aws-sdk-java-v2/issues/5577)
```

Reply:
Agreed. Do you want me to raise a PR to amend the changelog?

Reply:
Yeah, that'd be great. It may be easier after release is finished.